packages/lab/src/Flow/base.ts

```diff
@@ -1,10 +1,115 @@
 import { css } from "@emotion/react";
-import { theme } from "@hitachivantara/uikit-react-core";
```

Comment:
shouldn't we use our own tokens instead? eg. `theme.colors.*`, `theme.space.xs` instead of "10px", etc.